samples/ballerina-by-example/examples/salesforce-client-connector/salesforcerest-client-connector.bal

```diff
@@ -0,0 +1,63 @@
+import org.wso2.ballerina.connectors.salesforcerest;
```

Comment:
The names of the files and sample folder should be same. Here the folder name is "salesforce-client-connector" and files are named as "salesforcerest-client-connector.*". Both should be same